src/Faker/Provider/fa_AF/Internet.php

```diff
@@ -0,0 +1,282 @@
+<?php
+
+namespace Faker\Provider\fa_AF;
+
+class Internet extends \Faker\Provider\Internet
+{
+    /**
+     * @example 'a.sherzad'
+     * @example 'ab.sherzad'
+     * @example 'sherzad20'
+    */
+    protected static $userNameFormats = array(
+        '{{firstNameAsciiFirstChar}}.{{lastNameAscii}}',
+        '{{firstNameAsciiFirstChars}}.{{lastNameAscii}}',
+        '{{lastNameAscii}}##',
+    );
+
+    /**
+     * @example 'a.sherzad@yahoo.com'
+     * @example 'ab.sherzad@gmail.com'
+     * @example 'sherzad20@sherzad.af'
+    */
+    protected static $emailFormats = array(
+        '{{userName}}@{{domainName}}',
+        '{{userName}}@{{freeEmailDomain}}',
+    );
+
+    /**
+     * @example 'http://www.sherzad.com'
+     * @example 'http://www.ab.sherzad.af'
+     * @example 'http://www.sherzad.edu.af'
+    */
+    protected static $urlFormats = array(
+        'http://www.{{domainName}}',
+        'https://www.{{domainName}}',
+    );
+
+    protected static $freeEmailDomain = array(
+        'gmail.com', 'yahoo.com', 'hotmail.com', 'aol.com', 'outlook.com', 'mail.com'
+    );
+
+    /**
+     * @source The data collected by Abdul Rahman Sherzad (absherzad@gmail.com).
+    */
+    protected static $tld = array(
+        'com', 'gov', 'org', 'af', 'com.af', 'net', 'info', 'edu.af', 'gov.af',
+    );
+
+    /**
+     * @source Afghanistan Kankor (University Entrance Exam) 2003-2017 data.
+     * The data collected, preprocessed and prepared by Abdul Rahman Sherzad (absherzad@gmail.com).
+     * @license: Creative Commons Attribution-ShareAlike License
+    */
+    protected static $lastNameAscii = array("Ahmadi", "Mohammadi", "Noori", "Rahimi", "Amiri", "Haidari", "Azizi", "Siddiqi", "Hosseini", "Nazari", "Karimi", "Rahmani", "Hashemi",
```

Comment:
should be sorted alphabetically and split by line for each new capital letter (see formatters in other languages for a guide)